sentry/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 21.3.3
+version: 21.3.4
```

Comment:
Can you please upgrade the minor version?

Reply:
Updated to 21.4.0